tests/utils/test_db.py

```diff
@@ -0,0 +1,14 @@
+from _pytest.fixtures import fixture
+from page_analyzer.utils.db import create_url, get_url_data
+
+
+def test_create_url(test_database: fixture):
```

Comment:
fixtures are usually typed by the value they return - as test_database returns nothing, its type should be None